server2/handlers/like.handler.ts

```diff
@@ -0,0 +1,77 @@
+import {Request, Response} from "express"
+import {IMainController}   from "../controllers/main.controller";
+import * as dayjs          from "dayjs";
+import {IUser}             from "../../shared/types/Entities/IUser";
+
+
+// Get all likes
+export const getLikesHandler = async function (this: IMainController, req: Request, res: Response) {
+	try {
+		const likes = await this.likeController.getLikes()
+		return res.send(likes);
+
+	} catch (e) {
+		return res.status(404).send({msg: 'like has not found' + e})
+	}
+}
+
+// Create new like
+export const createLikeHandler = async function (this: IMainController, req: Request, res: Response) {
+	const
+		userLiked: IUser = await this.userController.getOneUser(req.body.user_id),
+		like             = {
+			timestamp: dayjs().format('DD.MM.YY'),
+			userLiked: userLiked,
```

Comment:
in es6 there is object property shorthand.  

```
like             = {
             			userLiked: userLiked
                       }
```
can be
like             = {
             			userLiked
                       }